test/test_coffeescript.rb

```diff
@@ -37,7 +37,8 @@ class TestCoffeeScript < JekyllUnitTest
     end
 
     should "write a JS file in place" do
-      assert_exist @test_coffeescript_file, "Can't find the converted CoffeeScript file in the dest_dir."
+      assert_exist @test_coffeescript_file, "Can't find the converted CoffeeScript file "\
+      "in the dest_dir."
```

Comment:
Let's simply remove the message here – I believe our `assert_exist` adds an acceptable message. 😄 

```ruby
assert_exist @test_coffeescript_file
```